### PR TITLE
Feat/adm 633 Frontend - replace date range styles 

### PR DESCRIPTION
**Pull request by K-Chow**
## Summary

...

## Before

_Description_

**Screenshots**
If applicable, add screenshots to help explain behavior of your code.

## After

_Description_

**Screenshots**
If applicable, add screenshots to help explain behavior of your code.

## Note

_Null_

### CODE REVIEW

File: frontend/__tests__/src/components/Common/DateRangeViewer/DateRangeViewer.test.tsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { render, screen } from '@testing-library/react'
+import DateRangeViewer from '@src/components/Common/DateRangeViewer'
+describe('DateRangeVier', () => {
+  it('should render component', () => {
+    render(<DateRangeViewer startDate={new Date('2022/1/1').toString()} endDate={new Date('2022/1/2').toString()} />)
```
Comment (guzhongren) [inline frontend/__tests__/src/components/Common/DateRangeViewer/DateRangeViewer.test.tsx:5]: new line for `given when then`

File: frontend/src/components/Common/DateRangeViewer/index.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+type Props = {
+  startDate: string
+  endDate: string
+}
+
+const DateRangeViewer = ({ startDate, endDate }: Props) => {
+  return (
+    <DateRangeContainer>
+      {dayjs(startDate).format(DATE_FORMATE_TEMPLATE)}
```
Comment (guzhongren) [inline frontend/src/components/Common/DateRangeViewer/index.tsx:13]: replace it with an extracted function.

File: frontend/src/components/Common/DateRangeViewer/style.tsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import styled from '@emotion/styled'
+import { ArrowForward, CalendarToday } from '@mui/icons-material'
+
+export const DateRangeContainer = styled.div({
+  display: 'flex',
+  justifyContent: 'flex-start',
+  alignItems: 'center',
+  backgroundColor: '#f3f3f3',
```
Comment (guzhongren) [inline frontend/src/components/Common/DateRangeViewer/style.tsx:8]: Can we use the defined value of `theme` to replace it. or and it into `theme`

Try to reduce the new color value

File: frontend/src/components/Common/DateRangeViewer/style.tsx [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  borderRadius: '0.4rem',
+  border: '0.07rem solid #d8d8d8',
+  width: 'fit-content',
+  padding: '.6rem',
+  color: '#7c7c7c',
+})
+
+export const StyledArrowForward = styled(ArrowForward)({
+  color: '#7c7c7c',
```
Comment (guzhongren) [inline frontend/src/components/Common/DateRangeViewer/style.tsx:17]: same issues as below

File: frontend/src/components/Common/DateRangeViewer/style.tsx [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+  margin: '0 .25rem',
+  fontSize: '1rem',
+})
+
+export const StyledCalendarToday = styled(CalendarToday)({
+  color: '#7c7c7c',
+  marginLeft: '.25rem',
+  fontSize: '1rem',
```
Comment (guzhongren) [inline frontend/src/components/Common/DateRangeViewer/style.tsx:25]: `1rem` is necessary? I think the default fontSize is `1rem`, expect you change its parent's fontSize and inherit it

File: frontend/src/components/Metrics/MetricsStep/style.tsx [hunk 6]
```diff
@@ -1,6 +1,12 @@
 import { styled } from '@mui/material/styles'
 import { Divider } from '@src/components/Common/MetricsSettingTitle/style'
 
+export const MetricSelectionHeader = styled('div')({
+  display: 'flex',
+  justifyContent: 'flex-end',
+  marginBottom: '1rem',
```
Comment (guzhongren) [inline frontend/src/components/Metrics/MetricsStep/style.tsx:7]: same issue as above

**Comment by K-Chow** [comment]
replace date range component for Metrics page

File: frontend/src/components/Common/DateRangeViewer/style.tsx (hunk 5 above)
Comment (K-Chow) [reply frontend/src/components/Common/DateRangeViewer/style.tsx:25]: actually, the MUI ICON component has a higher-priority style declaration setting the font size to 1.5rem. The 1rem here is to override the default style.

File: frontend/src/components/Common/DateRangeViewer/style.tsx (hunk 3 above)
Comment (K-Chow) [reply frontend/src/components/Common/DateRangeViewer/style.tsx:8]: Of course yes. just requires a bit of UX support.

**Comment by K-Chow** [comment]
1411703494019_ pic
1421703494036_ pic
1431703494046_ pic
1441703494056_ pic

**Review by SimonTal — APPROVED** [review]
Approved

**Pull request merged**